src/routes/auth.js

```diff
@@ -50,4 +53,71 @@ router.post('/signup', async (request, response, next) => {
   }
 });
 
+router.post('/forgot-password', async (request, response, next) => {
+  try {
+    if (
+      !request.body.email.trim() ||
+      !validator.isEmail(request.body.email.trim())
```

Comment:
I would reccomend updating this to the following - 
```javascript
if (!userEmail || !userEmail.trim() || !validator.isEmail(request.body.email.trim())
```
as the addition of 
```javascript 
!userEmail
``` 
checks if `userEmail` is either `undefined`, `null`, `0`, `false`, or an empty string ("").